docs/core-api/BOOTSTRAP.md

```diff
@@ -113,27 +162,64 @@ console.log(res.Peers)
 
 A great source of [examples][] can be found in the tests for this API.
 
-## `ipfs.bootstrap.rm([addr,] [options])`
+## `ipfs.bootstrap.rm(addr, [options])`
 
 > Remove a peer address from the bootstrap list
 
 ### Parameters
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| addr | [MultiAddr][] | The address of a network peer. If omitted the `all` option must be passed as `true`. |
+| addr | [MultiAddr][] | The address of a network peer |
```

Comment:
```suggestion
| addr | Array<MultiAddr> | The address of a network peer |
```
This is no longer optional, right ?

Reply:
Also a reference link.  But quite right, the note about `all` option should be removed.